src/web/components/SharingPermission/ParticipantSearchBar.scss

```diff
@@ -48,36 +38,36 @@
 
 .search-bar-dropdown {
   background: var(--theme-background-content);
-  box-shadow: 0px 4px 8px var(--theme-box-shadow);
-  border-radius: 0px 0px 10px 10px;
+  box-shadow: 0 4px 8px var(--theme-box-shadow);
+  border-radius: 0 0 10px 10px;
 }
 
 .search-bar-type-filter {
-  padding: 20px 20px 15px 20px;
+  padding: 20px 20px 15px;
   border-bottom: 1px solid var(--theme-table-border);
 }
 
 .search-bar-participants {
   padding: 20px;
-}
 
-.search-bar-participants th,
-td {
-  background: none !important;
-  padding: 0 0 1rem 0 !important;
-}
+  th,
+  td {
+    background: none !important;
+    padding: 0 0 1rem !important;
```

Comment:
I guess we could update padding here to use px?

Reply:
I've updated all the sizing to use px for layout and rem for fonts.